xgi/algorithms/assortativity.py

```diff
@@ -82,13 +82,24 @@ def degree_assortativity(H, kind="uniform", exact=False, num_samples=1000):
     float
         the degree assortativity
 
+    Raises
+    ------
+    XGIError
+        If there are no nodes or no edges
+
     References
     ----------
     Phil Chodrow,
     Configuration models of random hypergraphs,
     Journal of Complex Networks 2020.
     DOI: 10.1093/comnet/cnaa018
     """
+
+    if H.num_nodes == 0:
+        raise XGIError("Hypergraph must contain nodes")
+    elif H.num_edges == 0:
+        raise XGIError("Hypergraph must contain edges!")
+
```

Comment:
Can you make this consistent with the statement in the `dynamical_assortativity()` by copying these lines to that funcion or vice-versa?